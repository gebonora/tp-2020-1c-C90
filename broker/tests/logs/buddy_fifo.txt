Partición 1: | 0 - 15 | [L] | Size: 16 b | LRU: 9 |
Partición 2: | 16 - 31 | [X] | Size: 16 b | LRU: 13 | COLA: CATCH | ID: 4 |
Partición 3: | 32 - 63 | [X] | Size: 32 b | LRU: 19 | COLA: CATCH | ID: 6 |
