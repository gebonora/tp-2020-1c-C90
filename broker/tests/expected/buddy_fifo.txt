Partición 1: | 0 - 15 | [L] | Size: 16 b |
Partición 2: | 16 - 31 | [X] | Size: 16 b | LRU: 4 | COLA: CATCH | ID: 4 |
Partición 3: | 32 - 63 | [X] | Size: 32 b | LRU: 6 | COLA: CATCH | ID: 5 |
